scripts/create.ts

```diff
@@ -212,6 +215,7 @@ describe("${packageName}", () => {
     const tests = join(dir, "tests");
     const src = join(dir, "src");
 
+    //@ts-ignore
```

Comment:
The scripts are all run without a reference tsconfig, since those settings don't really matter so much... but ts complained here so I added these.